lms/djangoapps/courseware/views.py

```diff
@@ -739,6 +739,24 @@ def registered_for_course(course, user):
         return False
 
 
+def get_cosmetic_display_price(course, registration_price):
+    """
+    Return Course Price as a string preceded by correct currency, or 'Free'
+    """
+    currency_symbol = settings.PAID_COURSE_REGISTRATION_CURRENCY[1]
+
+    price = course.cosmetic_display_price
+    if registration_price > 0:
+        price = registration_price
+
+    if price:
+        # Translators: This may look like $50, while in some languages it may look like 50$.
```

Comment:
OK. Another fix would be to add an encoding to the file.

Generally for translators notes, you should only explain how this string will look, so it's not confusing. Eg, "Translators: This will look like '$50', where the currency symbol is a symbol such as '$' and the price is a numerical amount in that currency. Adjust this display as needed for your language."